.gitignore

```diff
@@ -1,3 +1,4 @@
 target/
 /pack/
 /cli-test/
+.idea/
```

Comment:
We usually don't put IDE specific things into the checked-in `.gitignore`. Put it in your global `.gitignore`?